component-samples/coding-components/templates/cdn/README.md

```diff
@@ -0,0 +1,5 @@
+# Coding components CDN template
+
+📁 **[Click here to download this directory as a ZIP file](https://download-directory.github.io?url=https://github.com/Esri/jsapi-resources/tree/main/component-samples/coding-components/templates/cdn)** 📁
+
+Since the ArcGIS Maps SDK for JavaScript and Calcite Components are dependencies of Coding Components they have to be loaded first. Coding Components will detect that it is used under AMD and will require its Maps SDK dependencies through AMD loading.
```

Comment:
```suggestion
Since the ArcGIS Maps SDK for JavaScript and Calcite Components are dependencies of Coding Components they have to be loaded first. Coding Components will detect that it is used under AMD and will require its Maps SDK dependencies using the AMD module loader.
```